.github/renovate.json

```diff
@@ -11,6 +12,23 @@
   "rebaseWhen": "conflicted",
   "semanticCommits": "disabled",
   "packageRules": [
+    {
+      "matchManagers": ["npm"],
+      "commitMessageTopic": "npm package `{{depName}}`",
+      "additionalBranchPrefix": "js/"
+    },
+    {
+      "enabled": true,
+      "groupName": "Block Protocol",
+      "matchManagers": ["npm"],
+      "matchPackagePatterns": [
+        "^@blockprotocol/",
+        "^block-",
+        "^blockprotocol$",
+        "^create-block-app$",
+        "^mock-block-dock$"
+      ]
+    },
```

Comment:
I just noted that `groupName` does not mean the packages are put in the same section but a PR for all these files will be created at once. In this case this is likely not what as we for example don't want to update all `@blockprotocol` at once.
```suggestion
```

Reply:
The PR is still approved, just hit `merge` if you disagree with me 🙂

Reply:
Why wouldn't we want to merge all Þ packages imported into the # repo at once? cc @CiaranMn

Reply:
If we keep on top of it, it should be fine to upgrade all BP packages at once, because they're mostly interconnected, and new releases of some will often just be upgrading the version of another one, in which case it makes sense to do them together.

If there are releases of independent packages (e.g. `@blockprotocol/hook` and `@blockprotocol/events`) then it becomes a pain because you have to do unrelated work in the same PR to fix breaking changes or whatever is required, and you might end up splitting it out.

I don't think it particularly matters either way. We can pick one approach and change it later if it does.

Reply:
This means, that we cannot use renovate anymore to bump the type-system package as this would require updating all other BP packages. Yes, most packages are connected, but not all.